### ISSUE TITLE
Without using Sentinel Hub for processing

### ISSUE THREAD

**Issue by jassantos** [sentinel-hub/sentinel2-cloud-detector#4]
I actually have downloaded Sentinel 2 scenes. Is it possible not to use the sentinel hub to download the Sentinel 2 data and just use the ones I am keeping? As I have seen in your examples, it uses the Sentinel Hub.

**Comment by AleksMat** [comment]
Hi jassantos,

it is possible to use downloaded data for classification. Sentinel Hub services in the examples only provide the required data. You can also read your data, resize all band images to the same size and join them together into one `numpy` array where bands are in the order `B01,B02,B04,B05,B08,B8A,B09,B10,B11,B12` (not all bands are used).

The cloud masks can then be calculated like this
```Python
cloud_detector = S2PixelCloudDetector(threshold=0.4, average_over=4, dilation_size=2)

cloud_probs = cloud_detector.get_cloud_probability_maps(my_band_images)
cloud_masks = cloud_detector.get_cloud_masks(my_band_images)
```

**Comment by azupanc** [comment]
In addition, I believe you need to divide the band values by 10000. The classifier expects the reflectances to be between 0 and 1.

**Comment by jassantos** [comment]
Thank you AleksMat and azupanc for your responses. I will look into it.

**Comment by lvhengani** [comment]
Is it possible to have an end-to-end example of how to use sentinel2-cloud-detector without using sentinel hub? I tried the suggestions above on a scene with clouds but the resulting cloud_mask has zeros only. The way I am creating the `my_band_images` 4-dimensional array is shown below:

~~~Python
my_band_images = np.zeros((n_scenes, y_size, x_size, z_size))
~~~ 

In my case I am using a single scene so the `n_scenes=1`,  `z_size=10` and `y_size`, `x_size`  are the ysize and xsize of a sentinel-2 band respectively and of-course all bands have been resampled to the same resolution and I am using 20m. I have downloaded multiple scenes with clouds to test and in all cases the resulting `cloud_masks` and the max=0 and min=0.
